templates/base.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE "html">
```

Comment:
The DOCTYPE declaration should be `<!DOCTYPE html>` without quotes. This ensures that the document is correctly interpreted as HTML5.